Apps/WebClient/src/NewClientApp/src/App.vue

```diff
@@ -30,6 +35,16 @@ const isCommunicationVisible = computed(
         !route.path.toLowerCase().startsWith(Path.PcrTest.toLowerCase())
 );
 
+const isFooterVisible = computed(
+    () =>
+        appStore.appError === undefined &&
```

Comment:
The check on `appError` can be removed, as that sort of error (from failing to retrieve config) now bypasses App.vue altogether and renders AppErrorView.vue instead.